README.md

```diff
@@ -15,8 +15,8 @@ This project provides the following custom skills:
 
 | Skill | Description | Type |Language | Environment |Deployment|
 | --- | ----------- | -------| ----------- | ----------- | ----------- |
-| [Azure Open AI Custom Skills](AOAICustomSkills/README.md) | This demonstrates custom skills to do things in AOAI deployments like text summarization and entity recognition via an API call to model hosted in Azure AI Studio | Text | ![python](https://img.shields.io/badge/language-python-orange) | ![functions](https://img.shields.io/badge/deploy-Functions-blue) | ARM Template |
-| [Azure AI Studio Entity Recognition](EntityRecognitionCustomSkill/README.md) | summarizes provided text via an API call to model hosted in Azure AI Studio | Text | ![python](https://img.shields.io/badge/language-python-orange) | ![functions](https://img.shields.io/badge/deploy-Functions-blue) | ARM Template |
+| [Azure AI Studio Open AI Custom Skills](AOAICustomSkills/README.md) | This demonstrates custom skills to do things in AOAI deployments like text summarization and entity recognition via an API call to an Open AI language model hosted in Azure AI Studio | Text | ![python](https://img.shields.io/badge/language-python-orange) | ![functions](https://img.shields.io/badge/deploy-Functions-blue) | ARM Template |
```

Comment:
Remove "this" to be consistent with other rows.

Reply:
done